pkg/identity/reserved.go

```diff
@@ -27,6 +27,17 @@ func AddReservedIdentity(ni NumericIdentity, lbl string) {
 	cacheMU.Unlock()
 }
 
+// AddReservedIdentityWithLabels is the same as AddReservedIdentityWithLabels
```

Comment:
I do hope so :smile: 
```suggestion
// AddReservedIdentityWithLabels is the same as AddReservedIdentity
```